pipeline/terraform0.12/modules/pipeline_service/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "service_name" {}
+variable "cluster_arn" {}
+variable "cluster_name" {}
+
+variable "subnets" {
+  type = list(string)
+}
+
+variable "aws_region" {}
+variable "namespace_id" {}
+variable "container_image" {}
+
+variable "secret_env_vars" {
+  type = map(string)
+}
+
+variable "secret_env_vars_length" {}
```

Comment:
All env_vars_length variables should be purged! Purged!

(This is one of the very nice things about terraform 0.12 that we get to ditch.)